packages/gatsby-tinacms-mdx/src/errors.ts

```diff
@@ -32,8 +32,8 @@ export const pageQuery = graphql\`
 \`
   `
 
-export const ERROR_MISSING_MDX_RAW_MARKDOWN =
-  'useMdxForm(mdx) Required attribute `rawMarkdownBody` was not found on `mdx` node.' +
+export const ERROR_MISSING_MDX_RAW_MDX =
```

Comment:
perhaps this should be `ERROR_MISSING_MDX_RAW_MDX_BODY`

Reply:
What about `ERROR_MISSING_RAW_MDX_BODY`?